packages/svelte-ux/src/routes/+layout.svelte

```diff
@@ -191,7 +163,8 @@
 
       <QuickSearch options={quickSearchOptions} on:change={(e) => goto(e.detail.value)} />
 
-      <div class="border-r border-primary-content/20 pr-2 grid items-center">
+      <div class="border-r border-primary-content/20 pr-2 grid grid-cols-2 items-center">
+        <LanguageSelect />
```

Comment:
YAY!  The beginning of showing a lot of this hard work in the docs (especially how the Date* components are not Intl.

I know we need to figure out the reactivity / setting issue, and UI wise I'll refine this before we publish to production (especially since we don't have room for another icon on mobile, and already need to reduce what we have.

Reply:
Yep, I leave you the hand now :)